craft_parts/executor/environment.py

```diff
@@ -151,6 +151,10 @@ def _get_global_environment(info: ProjectInfo) -> Dict[str, str]:
     global_environment = {
         "CRAFT_ARCH_TRIPLET": info.arch_triplet,
         "CRAFT_TARGET_ARCH": info.target_arch,
```

Comment:
Should we put a comment here noting it's pending deprecation?

Reply:
And similarly, should we warn users when they use a deprecated name in a part?

Reply:
Yes and yes, updating.